src/components/Button.jsx

```diff
@@ -1,18 +1,16 @@
 /* eslint-disable linebreak-style */
+/* eslint-disable arrow-body-style */
 /* eslint-disable react/prop-types */

```

Comment:
- [ ] As mentioned by the previous reviewer please do not disable the linters since they're there to help you